v2/src/main/scala/com/snowflake/snowpark/code_templates/udf_generator.txt

```diff
@@ -0,0 +1,13 @@
+(0 to 2).map { x =>
```

Comment:
1. move the template from `main/scala` to `main/generated`
2. try to change the file name to `*.scala`